app/(tabs)/index.tsx

```diff
@@ -1,21 +1,40 @@
-import { StyleSheet, View, Text } from 'react-native'
+import { StyleSheet, View, Text, TextInput, Button } from 'react-native'
 import { OrangeButton } from '@/components/OrangeButton'
 import ListElement from '@/components/ListElement'
 import Dropdown from '@/components/Dropdown'
 import { useState } from 'react'
+import { useAuth } from '@/providers/AuthProvider'
 import Colors from '@/constants/Colors'
+import { Link } from 'expo-router'
 
 export default function TabOneScreen() {
     const [selected, setSelected] = useState('Option 1')
     const [isOpen, setIsOpen] = useState(false)
-    console.log('TabOneScreen')
+    const { authState, setApiUrl } = useAuth()
+
+    const [ip, setIp] = useState('');
+
+    const handleChangeIp = async () => {
+        await setApiUrl(ip);
+        alert('Zmieniono adres IP');
+    };
+
```

Comment:
why do we need to check IP?

Reply:
It's just for testing - as long as the backend is not deployed and the computer can change ip, you don't need to change it in the code. - will be removed later